src/components/templates/ContentTemplate.tsx

```diff
@@ -338,6 +352,49 @@ const ContentTemplate = forwardRef<
             </ContentContainer>
           </>
         )
+      case 3:
+        return (
+          <>
+            <ContentContainer>
+              {isExistMark === 'exist' ? (
+                favorites.map((content, index) => {
+                  const linkUrl = `/otherscontent/${27}`
```

Comment:
We have to take care of this later !

Reply:
yeah it's just test link like dummy data